src/test/java/api/BoundWithTests.java

```diff
@@ -222,6 +222,32 @@ public void canRetrieveBoundWithItemByHoldingsRecordId() throws InterruptedExcep
   }
 
   @Test
+  public void canFetchBoundWithItemWithManyParts() throws InterruptedException, TimeoutException, ExecutionException
+  {
```

Comment:
Personally, I would use SneakyThrows to avoid the checked exceptions. And include the opening bracket on the same line as the end of the definition.

Reply:
All right, not familiar with SneakyThrows.

Reply:
It's an [annotation](https://projectlombok.org/features/SneakyThrows) provided by lombok that (kinda) turns checked exceptions into runtime exceptions.

I don't advocate for it in production code. In tests, where any uncaught exception fails the test, it can get rid of a bunch of noise in the tests.